### PR TITLE
Andes startup order fix

### PR DESCRIPTION
**Pull request by Asitha**
Remove AndesKernelBoot method calls from qpid code. This is done as a
step towards decoupling qpid code from Andes code.

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/transport/ConfigSynchronizer.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+package org.wso2.andes.kernel.transport;
+
+import org.wso2.andes.kernel.AndesBinding;
+import org.wso2.andes.kernel.AndesException;
+import org.wso2.andes.kernel.AndesExchange;
+import org.wso2.andes.kernel.AndesQueue;
+
+
+public interface ConfigSynchronizer {
```
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/transport/ConfigSynchronizer.java]: need a comment.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/virtualhost/QpidConfigSynchronizer.java [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+import org.wso2.andes.server.binding.BindingFactory;
+import org.wso2.andes.server.exchange.Exchange;
+import org.wso2.andes.server.queue.AMQQueue;
+import org.wso2.andes.server.queue.AMQQueueFactory;
+import org.wso2.andes.server.store.ConfigurationRecoveryHandler;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+public class QpidConfigSynchronizer implements ConfigurationRecoveryHandler.QueueRecoveryHandler, ConfigurationRecoveryHandler.ExchangeRecoveryHandler, ConfigurationRecoveryHandler.BindingRecoveryHandler {
```
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/virtualhost/QpidConfigSynchronizer.java]: comments , explain why the need to consolidate all interfaces.?

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/virtualhost/QpidConfigSynchronizer.java [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    @Override
+    public ConfigurationRecoveryHandler.BindingRecoveryHandler completeExchangeRecovery() {
+        return null;
+    }
+
+    //add the queue into internal qpid if NOT present
+    public void queue(String queueName, String owner, boolean exclusive, FieldTable arguments, ProtocolType protocolType) {
+
+
+        synchronized (this) {
```
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/virtualhost/QpidConfigSynchronizer.java]: why the need for synchronize?

File: modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/transport/ConfigSynchronizer.java (hunk 1 above)
Comment (Asitha) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/transport/ConfigSynchronizer.java]: Comment added

**Pull request merged**